python/runtime/pai/utils.py

```diff
@@ -22,3 +24,27 @@ def copyfileobj(source, dest, ak, sk, endpoint, bucket_name):
     bucket = oss2.Bucket(auth, endpoint, bucket_name)
     # overwrite if exists
     bucket.put_object_from_file(dest, source)
+
+
+def get_bucket(name, ak=None, sk=None, endpoint=None):
```

Comment:
Do not write `utils` files, name it like `oss_ops.py`

Reply:
Done. Renamed to be `python/runtime/pai/oss.py`